addon.gypi

```diff
@@ -92,7 +92,7 @@
           '-luuid.lib',
           '-lodbc32.lib',
           '-lDelayImp.lib',
-          '-l"<(node_root_dir)/$(ConfigurationName)/<(node_lib_file)"'
+          '-l"<(node_lib_file)"'
```

Comment:
$(ConfigurationName) is a MSBuild macro unknown for Ninja. So I changed it to `target_arch` variable. But this won't work with `--nodedir` option on Windows. As a result `node_lib_file` contains full path to the node.lib now. 

See comments in PR #964. Actually that request should be merged before this one.